.gitignore

```diff
@@ -4,6 +4,7 @@
 *.ipr
 classes/
 fork.properties
+/test.functional/env/
```

Comment:
I prefer specific / extra .gitignore in each module I mean creating a file `test.functional/.gitignore` because the root ignore is global and very often contains many entries. separate files gives better visibility

Reply:
done